### ISSUE TITLE
ansible playbook fails on RedHat 6

### ISSUE THREAD

**Issue by cehbrecht** [bird-house/ansible-wps-playbook#50]
... when playbook updates the pywps config it ends with the following failure:
```
ValueError: zero length field name in format
```

We probably need to use the latest version of ansible, which works with Python 2.7 or above.

**Comment by cehbrecht** [comment]
Upgrade ansible with:
```
$ sudo yum install python27
$ curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
$  python2.7 get-pip.py
$ pip --version
$ pip install ansible
$ ansible --version
ansible 2.7.2
```

**Comment by cehbrecht** [comment]
fixed by PR #52.
